File: C/file/file.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
	FILE *file;
	file = fopen("example.txt", "r");
	if (file == NULL) {
		perror("Error opening file");
		return EXIT_FAILURE;
	}
	char line[256];
	while (fgets(line, sizeof(line), file)) {
		printf("%s", line);
	}
	fclose(file);
	return EXIT_SUCCESS;
}

File: Rust/File/file.rs
use std::fs::File;
use std::io::{self, BufRead, BufReader};

fn main() -> io::Result<()>
{
    let file = File::open("example.txt")?;
    let reader = BufReader::new(file);
    for line in reader.lines() {
        let line = line?;
        println!("{}", line);
    }
    Ok(())
}